src/components/ContactUs/ContactUs.tsx

```diff
@@ -1,15 +1,18 @@
 import ContactUsInformation from './ContactUsInformation/ContactUsInformation'
 import FormContacUs from './FormContactUs/FormContactUs'
 import { ContentWrapper } from './ContactUs.styles'
-import SectionTitle from 'components/SectionTitle/SectionTitle'
 import Section from 'ui/Section/Section'
 import Container from 'ui/Container/Container'
+import H2 from 'ui/Titles/H2'
+import useIntlMessages from 'hooks/useIntlMessages'
 
 const ContactUs: React.FC = () => {
+  const intl = useIntlMessages()
+
   return (
     <Section id="contact-us" size="lg" type="margin">
       <Container size="xl" isfullwidth>
-        <SectionTitle size="lg" text="Contactanos" />
```

Comment:
delete styled component SectionTitle

Reply:
Other components outside of Contact Us are using it.